packages/flutter/lib/src/material/button.dart

```diff
@@ -153,39 +158,52 @@ class _RawMaterialButtonState extends State<RawMaterialButton> {
       ? (_highlight ? widget.highlightElevation : widget.elevation)
       : widget.disabledElevation;
 
-    return new Semantics(
-      container: true,
-      button: true,
-      enabled: widget.enabled,
-      child: new ConstrainedBox(
-        constraints: widget.constraints,
-        child: new Material(
-          elevation: elevation,
-          textStyle: widget.textStyle,
-          shape: widget.shape,
-          color: widget.fillColor,
-          type: widget.fillColor == null ? MaterialType.transparency : MaterialType.button,
-          animationDuration: widget.animationDuration,
-          child: new InkWell(
-            onHighlightChanged: _handleHighlightChanged,
-            splashColor: widget.splashColor,
-            highlightColor: widget.highlightColor,
-            onTap: widget.onPressed,
-            child: IconTheme.merge(
-              data: new IconThemeData(color: widget.textStyle?.color),
-              child: new Container(
-                padding: widget.padding,
-                child: new Center(
-                  widthFactor: 1.0,
-                  heightFactor: 1.0,
-                  child: widget.child,
-                ),
+    Widget result = new ConstrainedBox(
+      constraints: widget.constraints,
+      child: new Material(
+        elevation: elevation,
+        textStyle: widget.textStyle,
+        shape: widget.shape,
+        color: widget.fillColor,
+        type: widget.fillColor == null ? MaterialType.transparency : MaterialType.button,
+        animationDuration: widget.animationDuration,
+        child: new InkWell(
+          onHighlightChanged: _handleHighlightChanged,
+          splashColor: widget.splashColor,
+          highlightColor: widget.highlightColor,
+          onTap: widget.onPressed,
+          child: IconTheme.merge(
+            data: new IconThemeData(color: widget.textStyle?.color),
+            child: new Container(
+              padding: widget.padding,
+              child: new Center(
+                widthFactor: 1.0,
+                heightFactor: 1.0,
+                child: widget.child,
               ),
             ),
           ),
         ),
       ),
     );
+
+    if (widget.slopPadding != null) {
```

Comment:
This will need at least two tests: One that verifies that the bounds of the semantics node are actually increased by slop and another one to verify that a tap dispatched to the (for testing purposes unreasonably large) "slop" area is  actually handled by the `onPressed` handler.

Reply:
We don't specifically have tests for the RawMaterialButton, do you know if this is intentional?  Otherwise happy to add some

Reply:
I'm pretty sure the lack of testing is not intentional :)

Reply:
Added two basic tests to check that hit detection is expanded and semantics are expanded